main.tex

```diff
@@ -171,10 +172,8 @@ \subsection{Background and previous work}
 Leitgeb\cite{leitgeb08} defines a theory with inner and outer notions of truth and an outer notion of probability which, instead of requiring that $\vp\leftrightarrow\Tq{\vp}$, weakens the truth predicate and assigns the same probability to $\varphi$ and $True(\ulcorner\vp\urcorner)$.
 In another paper, Leitgeb\cite{leitgeb12b} extends his theory to two other systems which have outer truth, an inner truth predicate and an inner probability function. These two satisfy his 'probabilistic convention T', that is, $\Pq{\Tq{\vp}}=\Pq{\vp}$. His systems can be shown to satisfy our reflection principle, but have not been extended to expectations.
 
-dispensing with an inner notion of truth lets us focus more closely on the underlying difficulties of reflection in a probabilistic logic.
-
-We work purely with inner and outer probability and outer truth; dispensing with an inner notion of truth lets us focus more closely on the underlying difficulties of reflection in a probabilistic logic.
-We also derive an extension of our probabilistic framework which can coherently talk about expectations.
+We work purely with inner and outer probability and outer truth because dispensing with an inner notion of truth lets us focus on the underlying difficulties of reflection in probabilistic logic.
```

Comment:
This edit removes important context. 
"We work purely with inner and outer probability and outer truth; dispensing with an inner notion of truth lets us focus more closely on the underlying difficulties of reflection in a probabilistic logic.
We also derive an extension of our probabilistic framework which can coherently talk about expectations."
contains important hedging and contextual info.